app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailFragment.java

```diff
@@ -145,7 +130,6 @@ public void onRefreshComplete() {
 
     @Override
     public void showEvent(Event event) {
-        listener.onEventLoaded(event, false);
```

Comment:
There is no guarantee that when this implementation will change. It is not really great to depend on a fragment to trigger the event being loaded for the main presenter, because, suppose if we remove dashboard as first page or remove the event loading logic from t completely (maybe because analytics API has been added), we'll need to implement it anyway. IMO, this should be independent